### ISSUE TITLE
[CORE] Redirect /admin to /admin/

### ISSUE THREAD

**Issue by dignajar** [bludit/bludit#776]
After the bug fix would be nice to have a redirect 301 `/admin` to `/admin/`.
Check what could be the best option in performance.

**Comment by dignajar** [comment]
We need to rollback the changes to the previus, and fix this function https://github.com/bludit/bludit/blob/master/bl-kernel/url.class.php#L38

**Comment by dignajar** [comment]
@anaggh I did what you mentioned.

https://github.com/bludit/bludit/commit/2e33afda0cb47534ecbbd2a40a0970e038a7bff0

Also, I create this new rule for redirections, maybe in the future, we can add some routing defined by the user.

**Comment by anaggh** [comment]
Is this useful?
```
if ($url->slug()==ADMIN_URI_FILTER) {
	Redirect::url(DOMAIN_ADMIN);
}
```

**Comment by anaggh** [comment]
Thanks! :+1:

**Comment by dignajar** [comment]
Implemented in Bludit v3.0.0.
